design/machine-remediation-request.md

```diff
@@ -0,0 +1,142 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# machine-remediation-request
```

Comment:
I'm conflicted about including the word "request". Creating a Machine resource for example is equally a "request", as is the case for most things in a declarative API. What do you think about just calling this "Machine Remediation"?

Reply:
+1